hhs_facilities test dtype conflict: int vs float

Epidata 0.0.9 includes support for facilities, but for whatever reason the below failure didn't show up in the CI results when I subsequently merged [PR647](https://github.com/cmu-delphi/covidcast-indicators/pull/647). Not sure why it's parsing an int in one place and a float in another though 😞 

```
___________________________ TestPull.test_pull_data ____________________________

self = <test_pull.TestPull object at 0x7f1f9bc5f7c0>
pull_data_iteratively = <MagicMock name='pull_data_iteratively' id='139773734156800'>

    @patch("delphi_hhs_facilities.pull.pull_data_iteratively")
    def test_pull_data(self, pull_data_iteratively):
        pull_data_iteratively.return_value = [{"collection_week": 20201204,
                                               "total_beds_7_day_sum": 2360,
                                               "all_adult_hospital_beds_7_day_sum": -999999,
                                               "inpatient_beds_7_day_avg": -999999.0,
                                               "total_icu_beds_7_day_avg": np.nan,
                                               "total_staffed_adult_icu_beds_7_day_avg": 32.4}]
        output = pull_data()
        assert output.shape == (1, 7)  # 1 mock row, 6 mock + 1 new timestamp column
        # verify nans cast properly and timestamp added
>       pd.testing.assert_frame_equal(
            output,
            pd.DataFrame({"collection_week": [20201204],
                          "total_beds_7_day_sum": [2360],
                          "all_adult_hospital_beds_7_day_sum": [np.nan],
                          "inpatient_beds_7_day_avg": [np.nan],
                          "total_icu_beds_7_day_avg": [np.nan],
                          "total_staffed_adult_icu_beds_7_day_avg": [32.4],
                          "timestamp": [pd.Timestamp("2020-12-04")]}),
            check_names=False)
E       AssertionError: Attributes of DataFrame.iloc[:, 0] (column name="collection_week") are different
E       
E       Attribute "dtype" are different
E       [left]:  float64
E       [right]: int64

test_pull.py:63: AssertionError
```

--- a/hhs_facilities/tests/test_pull.py
+++ b/hhs_facilities/tests/test_pull.py
@@ -62,8 +62,8 @@ class TestPull:
         # verify nans cast properly and timestamp added
         pd.testing.assert_frame_equal(
             output,
-            pd.DataFrame({"collection_week": [20201204],
-                          "total_beds_7_day_sum": [2360],
+            pd.DataFrame({"collection_week": [20201204.],
+                          "total_beds_7_day_sum": [2360.0],
                           "all_adult_hospital_beds_7_day_sum": [np.nan],
                           "inpatient_beds_7_day_avg": [np.nan],
                           "total_icu_beds_7_day_avg": [np.nan],

